### PR TITLE
Reupload CarlosAlmRom Assigment4

### PR DESCRIPTION
**Pull request by CarlosAlmRom**

### CODE REVIEW

**Comment by anaigmo** [comment]
this PR has more changes than only the files for the assignment 4 :( please separate the changes into different PRs

**Comment by CarlosAlmRom** [comment]
changes corrected

File: Assignment4/CarlosAlmRom-23A280/task07.py
Comment (anaigmo) [inline Assignment4/CarlosAlmRom-23A280/task07.py:1]: missing rdflib alternatives for all exercices in this task

File: Assignment4/CarlosAlmRom-23A280/task07.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+# TO DO
+from rdflib import FOAF
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0")
+foaf = Namespace("http://xmlns.com/foaf/0.1/")
+
+q4 = prepareQuery(
+    '''SELECT DISTINCT ?person
+    WHERE {
+        ?person foaf:knows ns:RockySmith .
```
Comment (anaigmo) [inline Assignment4/CarlosAlmRom-23A280/task07.py:83]: list the name, not the resource!

**Comment by anaigmo** [comment]
Submitted and fixed, no more PRs allowed. The PR contains more file changes from other assignments, so it will not be merged

**Pull request closed without merging**